keps/sig-windows/5100-windows-dsr-and-overlay-support/README.md

```diff
@@ -0,0 +1,951 @@
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+For DSR, kube-proxy should resync HNS rules and start using DSR again.
+
+###### Are there any tests for feature enablement/disablement?
+
+<!--
+The e2e framework does not currently support enabling or disabling feature
+gates. However, unit tests in each component dealing with managing data, created
+with and without the feature, are necessary. At the very least, think about
+conversion tests if API types are being modified.
+
+Additionally, for features that are introducing a new API field, unit tests that
+are exercising the `switch` of feature gate itself (what happens if I disable a
+feature gate after having objects written with the new field) are also critical.
+You can take a look at one potential example of such test in:
+https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
+-->
+
+For overlay, no, because the feature requires the cluster to be configured for overlay networking mode and cannot be enabled on a per-node basis.
+
+For DSR, no, but they can be added.
+
+### Rollout, Upgrade and Rollback Planning
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+<!--
+Try to be as paranoid as possible - e.g., what if some components will restart
+mid-rollout?
+
+Be sure to consider highly-available clusters, where, for example,
+feature flags will be enabled on some API servers and not others during the
+rollout. Similarly, consider large clusters and how enablement/disablement
+will rollout across nodes.
+-->
+
+For DSR a rollout or rollback should not fail. Nodes can operator with DSR enabled or disabled per node in a cluster.
+
+For overlay networking mode support, a rollout can fail if the CNI configuration for the node and kube-proxy configuration are not in sync. This would cause nodes to never go into the Ready state.
+
+###### What specific metrics should inform a rollback?
+
+<!--
+What signals should users be paying attention to when the feature is young
+that might indicate a serious problem?
+-->
+
+Node ready state should be monitored to ensure nodes job the cluster and are properly configured to start running pods.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+<!--
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+-->
+
+For DSR support yes, manual verification was done to ensure that DSR can be enabled and disabled on a node.
```

Comment:
Please describe the manual testing performed, see https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/3850-backoff-limits-per-index-for-indexed-jobs/README.md#were-upgrade-and-rollback-tested-was-the-upgrade-downgrade-upgrade-path-tested as an example.

Reply:
updated - PTAL